### ISSUE TITLE
link dead

### ISSUE THREAD

**Issue by 3052** [fayer3/plugin.video.joyn_app#9]
https://github.com/fayer3/plugin.video.joyn_app/blob/52f067c77a6c2005adc6ebfbaf0bbfc798b3d2ef/resources/lib/ids.py#L33-L34

**Comment by 3052** [comment]
<https://joyn.de/_next/static/chunks/3806.dcb7578b32019150.js>

but obfuscated:

~~~js
t[i(486) + "t"] = "5C783" + i(628) + "78646" + i(642) + i(701) + "65C78" +
i(545) + i(371) + i(336) + i(470) + "65395" + i(415) + i(438) + i(587) + i(468)
+ i(395) + i(561) + "5"
~~~

**Comment by fayer3** [comment]
yeah, not sure I'll do any updates soon since knaerzches addon works fine
but thanks for pointing me to the new thing

**Comment by 3052** [comment]
will you archive the repo then?

**Comment by fayer3** [comment]
not sure, maybe I'll update it at some point

**Comment by 3052** [comment]
where is knaerzches addon?

**Comment by 3052** [comment]
ah here

https://github.com/knaerzche/plugin.video.joyn
